harbor-cluster/templates/issuer.yaml

```diff
@@ -1,7 +1,19 @@
 # Cert issuer
 apiVersion: cert-manager.io/v1
-kind: Issuer
+kind: {{ .Values.core.ingress.issuer.kind }}
 metadata:
-  name: selfsigned-issuer
+  name: {{ .Values.core.ingress.issuer.name }}
 spec:
```

Comment:
Instead of having ACME, I would rather parametrize the whole spec of the issuer in the values file.

Reply:
ok